src/MonacoContainer/MonacoContainer.tsx

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
 
-import Loading from '../Loading';
-
-import styles from './styles';
+import styles from "./styles";
+import Loading from "../Loading";
 
 // ** forwardref render functions do not support proptypes or defaultprops **
 // one of the reasons why we use a separate prop for passing ref instead of using forwardref
 
+type ContainerProps = {
```

Comment:
so, currently our file/folder structure follows to this pattern:

```
ComponentName/
  - [ComponentName].tsx
  - index.ts
  - index.spec.tsx // I would change this to just spec.tsx,  but this is also okay
  - types.tsx
  // we also can have utils.ts, styles.ts, etc.
```

Let's extract this type declaration from here and move it to a separate `types.ts` file. Also, let's keep consistency and for component props type declaration use only `type` or `interface`. Currently, `Editor` and `DiffEditor` use `interface`, and `MonacoContainer` `type`. In all my projects I use `type`, but don't see any issue with `interface`, so, feel free to chose whatever you want

Reply:
Good shout. updated